docs/guides/component-testing/angular/examples.mdx

```diff
@@ -193,6 +193,192 @@ in the future
 
 :::
 
+### Testing Signals
+
+With the releases of Angular versions [17.1](https://github.com/angular/angular/blob/main/CHANGELOG.md#1710-2024-01-17) and [17.2](https://github.com/angular/angular/blob/main/CHANGELOG.md#1720-2024-02-14), [input](https://github.com/angular/angular/pull/53521) and [model](https://github.com/angular/angular/pull/54252) signals were introduced into the `@angular/core` API.
+Since signals introduced new methods and types to the core API, Cypress needed to introduce a new test harness, `@cypress/angular-signals`, in order to support the API without breaking changes.
```

Comment:
```suggestion
Since signals introduced new methods and types to the core API, Cypress introduced a new test harness, `@cypress/angular-signals`.
```

Reply:
addressed in 1055c7068